lib/spell-check-rules.js

```diff
@@ -21,8 +25,16 @@ module.exports = [
 		test: /\bYou\s?Tube\b/gi,
 		value: 'YouTube'
 	},
+	{
+		test: /\bYou\s?Tube\.\b/gi,
+		value: 'youtube.'
+	},
 	{
 		test: /\bGit\s?Hub\b/gi,
 		value: 'GitHub'
```

Comment:
Old regular expressions are still there.

Reply:
I think the old regexes are there because we're now differentiating between normal text and being part of a domain.

Reply:
@transitive-bullshit Those regular expressions work like a blacklist. Once matched, an error will be thrown. And no break in the midway, which means others will continue to be executed. Now we added a new expression to let cases like `github.com` to be passed, but the old one will still throw an error, due to it is also matched.